frontend/src/app/side-menu/side-menu.component.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { LayoutModule } from '@angular/cdk/layout';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { MatButtonModule } from '@angular/material/button';
+import { MatIconModule } from '@angular/material/icon';
+import { MatListModule } from '@angular/material/list';
+import { MatSidenavModule } from '@angular/material/sidenav';
+import { MatToolbarModule } from '@angular/material/toolbar';
+
+import { SideMenuComponent } from './side-menu.component';
```

Comment:
How are the imports being sorted and organized?
Recommend split into two with a comment line:
/* Angular imports. */
...
/* Project imports. */

Reply:
& sort within each group preferably by the module from which components are being imported

Reply:
Thanks for the suggestion. I couldn't find specific Angular import style guidelines so I'll follow that suggestion

Reply:
This file is autogenerated so I left it as is.